include/zephyr/net/net_context.h

```diff
@@ -1223,6 +1223,10 @@ int net_context_recv(struct net_context *context,
 		     k_timeout_t timeout,
 		     void *user_data);
 
+int net_context_recvfrom(struct net_context *context,
```

Comment:
The function prototype does not differ from regular `net_context_recv()`, how would be the address provided in such case?

Reply:
I have removed this in favor to add peer IP in struct net_pkt.